src/javascript/events/component-view.js

```diff
@@ -0,0 +1,67 @@
+/*global module, require */
+'use strict'; // eslint-disable-line strict
+
+const Core = require('../core');
+const getTrace = require('../../libs/get-trace');
+const utils = require('../utils');
+
+const decorateEventData = (eventData, viewedEl, opts) => {
+
+	const { trace, customContext } = getTrace(viewedEl);
+	let context = typeof opts.getContextData === 'function' ? opts.getContextData(viewedEl) : {};
```

Comment:
I'm wondering if it's worth running the object returned from `getContextData` through a whitelist like [o-audio](https://github.com/Financial-Times/o-audio/blob/master/src/js/tracking.js#L46). In practice these attributes need to be agreed with the data team + would provide some free documentation as to what can be returned. What ya think?

Reply:
I see! I will do that. Thank you!